graph/api/src/main/java/org/jboss/windup/graph/model/ProjectModel.java

```diff
@@ -101,18 +106,32 @@
     void setVersion(String version);
 
     /**
-     * Contains the project's name.
+     * Project's name. This is most often derived from project's root filename.
```

Comment:
Fixed.
     * Project's name.
     * This is often derived Maven name or MANIFEST.MF name or from project's root filename.